app/config/config.yml

```diff
@@ -58,6 +58,7 @@ framework:
         resource: '%kernel.root_dir%/config/routing.yml'
     secret: '%secret%'
     session:
+        cookie_lifetime: 86400 # 1 day
```

Comment:
A bit unrelated, but while until Profiles is a bit more developed (ie fetches details from ORCID), we should probably limit the lifetime. We'll need to set it normally, as I think the default is forever, which will never disappear from Redis.

Reply:
:+1: for short when unstable, longer when will be stable